packages/bratus-app/src/api/index.js

```diff
@@ -2,8 +2,13 @@ export const getParsedData = () => {
   return fetch('http://localhost:4444/parsedData').then((res) => res.json());
 };
 
-export const recompile = () => {
+export const recompile = (input) => {
+  console.log('[ASTParser] Recompiling...');
   return fetch('http://localhost:4444/compile', {
     method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({ rootComponents: input }),
```

Comment:
input parameter can also be changed to rootComponents or rootsFromInput or smth else?